tests/test_themes.py

```diff
@@ -0,0 +1,51 @@
+from flask import current_app
+import pytest
+
+
+themes = [
+    'cerulean',
+    'cosmo',
+    'cyborg',
+    'darkly',
+    'default',
+    'flatly',
+    'journal',
+    'litera',
+    'lumen',
+    'lux',
+    'materia',
+    'minty',
+    'pulse',
+    'sandstone',
+    'simplex',
+    'sketchy',
+    'slate',
+    'solar',
+    'spacelab',
+    'superhero',
+    'united',
+    'yeti'
+]
+
+
+class TestThemes:
+    @pytest.mark.parametrize("theme", themes)
```

Comment:
In this line and related lines, the single quotation marks should be used in order to consistent with others.

Reply:
Thanks for pointing that out, i have updated it on the new commit